### PR TITLE
Use custom boilerplate for src/lib.rs

### PR DESCRIPTION
**Pull request by maackle**
TODO: actually write the boilerplate
For this, follow up issue #25, we'll leave it open, for once stuff is better defined early next week

### CODE REVIEW

File: src/cli/scaffold/rust/lib.rs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#[macro_use]
+extern crate hdk;
+
+// TODO: make this a nice friendly  boilerplate
```
Comment (Connoropolous) [inline src/cli/scaffold/rust/lib.rs]: I'm going to add the issue number here, which is a pattern we're trying to keep up. Then, we can leave that issue open, to actually fulfill this boilerplate

**Comment by maackle** [comment]
Should we even bother merging this until we have a proper boilerplate? I would argue that the autogenerated lib.rs is more useful than this one!

**Comment by Connoropolous** [comment]
Fixed this @sphinxc0re

**Comment by Connoropolous** [comment]
@maackle I had the idea to provide good code examples somewhere else, and then link to those, from the lib.rs scaffold file

**Comment by Connoropolous** [comment]
if someone could please merge this one it'd be great. @lucksus @zippy

**Pull request merged**